Directory.Build.props

```diff
@@ -12,11 +12,13 @@
     <IncludeSymbols>true</IncludeSymbols>
     <IsShippingPackage>false</IsShippingPackage>
     <SdkTargetFramework>net8.0</SdkTargetFramework>
+    <!-- Temporarily disable build warnings while upgrading to the net9.0 TFM -->
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
Why is that needed? That switch isn't intended to be set by our own repositories. It's a last effort escape switch.

Reply:
The build was throwing a massive number of errors about incompatible runtime targeting. In prior years, we have not had to retarget the SDK before merging in a new runtime so this seems to be a new error:

Reply:
So adding that flag was to suppress the error so we could get runtime merged and then retarget later (last year, retargetting happened ~3 months after rebranding)

Reply:
dotnet/runtime retargeting happen soon after rebranding this year while that happened around December last year. That's probably the reason why you weren't impacted by this.

IMO it makes sense to do the retargeting here as part of consuming the new runtime. IIRC that's what we did in the previous releases, before .NET 7. But if this switch is just here for a few weeks, then I'm fine with that.

Reply:
ahh, I didn't realize runtime was that late last time around. That explains it. I'd like to take this whatever the shortest path to fixing the failures is